ui/src/organizations/components/VariableRow.tsx

```diff
@@ -11,10 +11,13 @@ import {
 
 // Types
 import {Variable} from '@influxdata/influx'
+import EditableName from 'src/shared/components/EditableName'
 
 interface Props {
   variable: Variable
   onDeleteVariable: (variable: Variable) => void
+  onUpdateVariable: (variable: Partial<Variable>) => void
+  onEditVariable: (variable: Variable) => void
```

Comment:
perhaps there is a better name distinction between these two props? both of those names seem like they would do the same thing.